packages/compiler-core/__tests__/testUtils.ts

```diff
@@ -17,7 +17,7 @@ const bracketsRE = /^\[|\]$/g
 // e.g.
 // - createObjectMatcher({ 'foo': '[bar]' }) matches { foo: bar }
 // - createObjectMatcher({ '[foo]': 'bar' }) matches { [foo]: "bar" }
-export function createObjectMatcher(obj: any) {
+export function createObjectMatcher(obj: AnyObject) {
```

Comment:
```suggestion
export function createObjectMatcher(obj: any) {
```

Reply:
Parameter `obj` can not be base type, I think use `any` type here is not rigorous, at least should use `object`.
However, if you use `object`, function `createObjectMatcher` will prompt an Error that call `No index signature`, so here I think it would be more appropriate to use the following types:
```
Interface AnyObject {
     [key: string]: any;
}
```
If you think that the global type is not very good, I can write this above the `createObjectMatcher`.

Reply:
can't you assign to `object` or to a `Record<string, any>`?

Reply:
Good idea

Reply:
> can't you assign to `object` or to a `Record<string, any>`?

`Record<string, any>` is a good idea, I forgot it before.